mitodl/frontend-component-footer-mitol
# fix: fix config name from MARKETING_BASE_URL to MARKETING_SITE_BASE_URL

# What are the relevant tickets?




Subtask of https://github.com/mitodl/hq/issues/3135

# Description (What does it do?)

The config name for the marketing URL is MARKETING_SITE_BASE_URL but MARKETING_BASE_URL is used in the Footer and it is not getting picked up from the config. This PR just fixes the name of the config variable.

# How can this be tested?


- Setup footer and authoring MFE
- Add MARKETING_SITE_BASE_URL in authoring MFE .env.development
- Verify the link the authoring MFE footer 

# Screenshot
![Screen Shot 2023-12-22 at 5 56 57 PM](https://github.com/mitodl/frontend-component-footer-mitol/assets/52656433/7de05ef4-be24-4be3-8ac6-6b5eb3deebc1)
